deploy/playbooks/deploy.yml

```diff
@@ -14,4 +14,5 @@
      ansible_ssh_port: 2222
      binary_root: "/opt/binaries"
      repos_root: "/opt/repos"
+     distributions_root: "/opt/distributions"
```

Comment:
why instead of this we don't include that dir in the actual project, so it becomes `%(confdir)s/distributions`? I think that would be even better than `/etc/distributions/*`